tests/baselines/reference/stringLiteralTypesInUnionTypes04.types

```diff
@@ -81,9 +81,9 @@ if (!!!x) {
 >!!!x : boolean
 >!!x : boolean
 >!x : boolean
->x : T
+>x : "" | "foo"
 
     let h = x;
->h : T
->x : T
+>h : "" | "foo"
+>x : "" | "foo"

```

Comment:
Isn't this the opposite of the intended change?

Reply:
Yes, that's not intended. Fixed by latest commits.